### ISSUE TITLE
folder still shows 0 bytes after a rescan

### ISSUE THREAD

**Issue by marrie** [syncthing/syncthing#2230]
When adding a folder, the size still shows 0 bytes. I have the folders set up thus. I did  a rescan of the folder just to be sure. I should note this folder already exists.

 Folder Path	/Volumes/music/3archives/movies\ and\ tv/tv/sports
 Global State	0 items, ~0 B
 Local State	0 items, ~0 B
 Folder Master	Yes
 Rescan Interval	120 s
 File Pull Order	Largest First

This folder has subfolders as well as stuff at the root of said folder. I'm on a mac running the latest version of Yosemite

A suggestion might be to have a browse button  next to the folder path that way we don't have to type in the path manually.

Attached is a link to a screenshot of this same area.

https://www.dropbox.com/s/4x7tp2j97dadtgk/Screenshot%202015-09-03%2012.33.13.png?dl=1

**Comment by wweich** [comment]
Did you try restarting syncthing, as it is telling you to do so?

**Comment by marrie** [comment]
Yep. First thing I did, three times. No joy.

**Comment by uok** [comment]
try entering the path without the "\" before spaces in the path and restart Syncthing
`/Volumes/music/3archives/movies and tv/tv/sports`
Syncthing also has an autosuggest feature while you type

**Comment by AudriusButkevicius** [comment]
Why do you even have the \ to start with? Did you copy the path from bash or something?

**Comment by marrie** [comment]
To answer the question of why I had backslashes, I always include them
if I am unsure if a program will auto escape a character like a space.
I had to actually re-add the folder as the edit section would not let
me edit the folder in question. It appears to work now. time will
tell. I saw the right number of items and the space looked a bit
small. This folder was 200 gigs in size and it was  showing 150
thousand  kBytes for the 178 thousand odd items in there.  I believe
that's correct though.
